Source/Svg.csproj

```diff
@@ -1,102 +1,151 @@
+            So please feel free to fork it and open pull requests for any fix, improvement or feature you add.
+            License: Microsoft Public License.
+        </Description>
+        <Copyright>Copyright © vvvv.org</Copyright>
+        <Tags>svg, vector graphics, rendering</Tags>
+        <ProjectGuid>{886A98C5-37C0-4E8B-885E-30C1D2F98B47}</ProjectGuid>
+        <Configurations>Debug;Release</Configurations>
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>svgkey.snk</AssemblyOriginatorKeyFile>
+        <PackageTags>svg, vector graphics, rendering</PackageTags>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <PackageReleaseNotes>
+            Fixed a regression in loading embedded images (see Release Notes: http://svg-net.github.io/SVG/doc/ReleaseNotes.html).
+        </PackageReleaseNotes>
+        <PackageProjectUrl>https://github.com/svg-net/SVG</PackageProjectUrl>
+        <PackageIconUrl>https://www.w3.org/Icons/SVG/svg-logo-v.png</PackageIconUrl>
+        <PackageIcon>svg-logo-v.png</PackageIcon>
+        <LangVersion>10.0</LangVersion>
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+        <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+        <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\$(Configuration)\$(TargetFramework)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
+    </PropertyGroup>
+    
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+        <DebugType>full</DebugType>
+        <DebugSymbols>true</DebugSymbols>
+        <DefineConstants>DEBUG;TRACE</DefineConstants>
+        <NoWarn>$(NoWarn);CS0618;NETSDK1138</NoWarn>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+        <DebugType>portable</DebugType>
+        <Optimize>true</Optimize>
+        <DefineConstants>TRACE</DefineConstants>
+        <NoWarn>$(NoWarn);CS1591;CS0618;NETSDK1138</NoWarn>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    </PropertyGroup>
+    
+    <PropertyGroup>
+        <DefineConstants Condition="$(TargetFramework.StartsWith('net46'))">$(DefineConstants);DOTNET46;NETFULL</DefineConstants>
+        <DefineConstants Condition="$(TargetFramework.StartsWith('net47'))">$(DefineConstants);DOTNET47;NETFULL</DefineConstants>
+        <DefineConstants Condition="$(TargetFramework.StartsWith('net48'))">$(DefineConstants);DOTNET48;NETFULL</DefineConstants>
```

Comment:
Why are these needed?

Reply:
Added support for .NET Framework 4.7 and 4.8, for project using them.
The constants are provided in case any platform specific coding is necessary. Currently, only `NETFULL` is used.

Reply:
Ok, as long as there are no additional projects are build by default, it won't actually matter, so it is ok, I guess.